mne_features/feature_extraction.py

```diff
@@ -0,0 +1,81 @@
+import warnings
```

Comment:
missing authors and license on top of file

Reply:
Fixed.